templates/workspace_services/guacamole/porter.yaml

```diff
@@ -126,6 +126,7 @@ install:
         guac_disable_download: "{{ bundle.parameters.guac_disable_download }}"
         api_client_id: "{{ bundle.parameters.api_client_id }}"
         aad_tenant_id: "{{ bundle.parameters.aad_tenant_id }}"
+        key: "{{ bundle.parameters.id }}"
```

Comment:
I think the name "key" can be renamed to something more meaningful for terraform in this case. What about something like tre_resource_id?

Reply:
Fixed